README.md

```diff
@@ -170,6 +170,15 @@ Apache Version 2.0
 
 See [LICENSE](https://github.com/googleapis/nodejs-logging-winston/blob/master/LICENSE)
 
-[client-docs]: https://cloud.google.com/logging/docs/setup/nodejs
+[client-docs]: https://cloud.google.com/nodejs/docs/reference/logging-winston/latest/
```

Comment:
target page is 404 ATM. Expected?

Reply:
Yes, I can't publish those docs until this PR is merged and a new release of this library is published.